### PR TITLE
Create room type

### PR DESCRIPTION
**Pull request by at-tringuyen3**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Task ID: TI-15

## Reference Isuees
#15

## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: app/Http/Controllers/RoomTypeController.php [hunk 1]
```diff
@@ -50,9 +53,15 @@ public function create()
...
      * @return \Illuminate\Http\Response
      */
-    public function store(Request $request)
+    public function store(RoomTypeRequest $request)
     {
-        echo $request;
+
+        $check = $this->roomType->addRoomType($request);
+        if ($check === true) {
+            return $this->redirectSuccess('room-types.index', 'Create Success');
```
Comment (at-tuannguyen3) [inline app/Http/Controllers/RoomTypeController.php]: define message trong localization luôn nhé

File: app/Http/Requests/admin/RoomTypeRequest.php [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    /**
+     * Config form return validation.
+     *
+     * @return array
+     */
+    public function messages()
+    {
+        return
+        [
+            'name.required' => 'Fill in the field',
```
Comment (at-tuannguyen3) [inline app/Http/Requests/admin/RoomTypeRequest.php]: dùng localization luôn nhé

File: app/Http/Controllers/RoomTypeController.php [hunk 3]
```diff
@@ -50,9 +53,15 @@ public function create()
...
      */
-    public function store(Request $request)
+    public function store(RoomTypeRequest $request)
     {
-        echo $request;
+        $data = $request->only('name');
+        $data['user_id'] = Auth::user()->id;
+        $check = $this->roomType->addRoomType($data);
+        if (!empty($check)) {
+            return $this->redirectSuccess('room-types.index', 'Create Success');
```
Comment (at-tuannguyen3) [inline app/Http/Controllers/RoomTypeController.php]: dùng localization cho message luôn nhé

**Review by ghost — CHANGES_REQUESTED** [review]
check comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**